superset/views/core.py

```diff
@@ -1085,8 +1085,14 @@ def save_dash(  # pylint: disable=no-self-use
         DashboardDAO.set_dash_metadata(dash, data)
         session.merge(dash)
         session.commit()
+
+        # get updated changed_on
+        dash = session.query(Dashboard).get(dashboard_id)
+        last_modified_time = dash.changed_on.replace(microsecond=0).timestamp()
         session.close()
-        return json_success(json.dumps({"status": "SUCCESS"}))
+        return json_success(
+            json.dumps({"status": "SUCCESS", "last_modified_time": last_modified_time,})
```

Comment:
Can we rename `last_modified_time` everywhere to `changed_on` if it's not too big of a change?

Having some consistency is nice.

Reply:
changed_on is a column name in Dashboards table, its type is `DATETIME`. I prefer to use timestamp so that ignore local time etc (maybe different setup in each company).